### PR TITLE
feat: add participating info statutory

### PR DESCRIPTION
**Pull request by LeonVreling**
back-end support for [#279 of oms-frontend](https://github.com/AEGEE/oms-frontend/pull/279). Still needs test to back up validity

### CODE REVIEW

**Comment by LeonVreling** [comment]
It's complaining a lot about the `merge()` in `merge.js`, which was copy-pasted from from `oms-events/lib/merge.js`. Is there a way to easily fix this, or should the function just be updated?

**Comment by LeonVreling** [comment]
Oh and also @serge1peshcoff , I can't find the button to request a review 😅

File: lib/server.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ GeneralRouter.get('/', events.listEvents);
 GeneralRouter.use(middlewares.ensureAuthorized);
 GeneralRouter.post('/', events.addEvent);
 GeneralRouter.get('/tasks', middlewares.getTasksList);
+GeneralRouter.get('/mine/participating', events.listUserAppliedEvents);
```
Comment (serge1peshcoff) [inline lib/server.js]: I suggest renaming it to `/events/mine`, oms-events has slightly different endpoints structure, there's no events list you are organizing

**Comment by serge1peshcoff** [comment]
@LeonVreling  here you can ask for a review image

also in oms-statutory there's `'deep-assign'` package, check `lib/config.js` for using it.

**Comment by LeonVreling** [comment]
Yeah it just did not give me that option with this PR

**Comment by LeonVreling** [comment]
Hm and also, my local copy does not have the `oms-statutory/lib/config.js` that you are referring to.

**Comment by serge1peshcoff** [comment]
@LeonVreling try `oms-statutory/config/index.js`, I probably misspelled it.
Also can you write some tests? You can check out `test/api` folder.
Basically 2 tests needed: that it displays the event you've applied to and that it doesn't display the event you haven't applied to.

**Comment by LeonVreling** [comment]
Hm I took a look at `deepAssign`, but it does not seem to be working as I constantly get an error that it can't fetch the statutory events list.

**Comment by LeonVreling** [comment]
Now it’s only failing audit 😁

**Pull request merged**